<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlAllProfilesWindow</class>
 <widget class="QMainWindow" name="ControlAllProfilesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="2" column="0">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="gb_account">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>Account's</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btn_create_account">
         <property name="text">
          <string>Create</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_read_account">
         <property name="text">
          <string>Read</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_update_account">
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_delete_account">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1" rowspan="7">
     <widget class="QTabWidget" name="tab_table">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>200</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_account">
       <attribute name="title">
        <string>Акаунти</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTableView" name="lv_account"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_education">
       <attribute name="title">
        <string>Стажування</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QListView" name="lv_education"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_resume">
       <attribute name="title">
        <string>Резюме</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QListView" name="lv_resume"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_vacancy">
       <attribute name="title">
        <string>Вакансії</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QListView" name="lv_vacancy"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="btn_logout">
      <property name="text">
       <string>Log out</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="btn_to_main_menu">
      <property name="text">
       <string>Back to main menu</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QGroupBox" name="gb_record">
      <property name="title">
       <string>Record's</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="btn_create_record">
         <property name="text">
          <string>Create</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_read_record">
         <property name="text">
          <string>Read</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_update_record">
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_delete_record">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>618</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuProfile">
    <property name="title">
     <string>Profile</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuWindow"/>
   <addaction name="menuProfile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
